_pytest/python.py

```diff
@@ -1104,8 +1104,7 @@ def raises(expected_exception, *args, **kwargs):
     ``message`` to specify a custom failure message::
 
         >>> with raises(ZeroDivisionError, message="Expecting ZeroDivisionError"):
-        ...    pass
-        ... Failed: Expecting ZeroDivisionError
+        ...    1/0
```

Comment:
It was incorrect before: it was meant to demonstrate an error, but the with block was just "passing". We never noticed because nobody has been running `pytest --doctest-modules`.